docs/attributes-registry/gen-ai.md

```diff
@@ -30,6 +30,13 @@ This document defines the attributes used to describe telemetry in the context o
 
 `gen_ai.system` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
 
-| Value    | Description | Stability                                                        |
-| -------- | ----------- | ---------------------------------------------------------------- |
-| `openai` | OpenAI      | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| Value         | Description  | Stability                                                        |
+| ------------- | ------------ | ---------------------------------------------------------------- |
+| `openai`      | OpenAI       | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `microsoft`   | Microsoft    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
we should have a specific service here, but are there any MS-owned GenAI solutions? Where would someone set it?

Reply:
Microsoft has its own series of models. I think we need a clear definition of `model provider`. 

Soon, there will be model as a service on Azure where users can run any models. Do we want all of those to be under Azure? In fact, you can already run Llama2 on Azure, do we want to categorize Llama2 under Azure?

Reply:
we'd probably have a constant for `maas` since it's the API provides that serves requests and proxies things to the actual model. 
It's the GenAI product client application interacts with. 

The name here should be the name of the GenAI product, not a generic company name. 
I'd suggest to remove Microsoft for now unless MaaS is already being instrumented.